app/views/landing/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= content_tag(:div, "Logotype", id: ["logotype"]) %>
```

Comment:
According to the finders dox that's the way to do it, but removing them works fine too. =)